### DISCUSSION TITLE
ssh-fs: Could not connect to React dev server. Not running?

### DISCUSSION THREAD

**Discussion by wieluk** [coder/code-server#3764] [Q&A]
I updated the addon now I when I try to add a new connection or change settings or current connections this pops-up. Does anyone has an idea? I could not find anything while googling. I also reverted back to the version I had when it worked but still getting "Could not connect to React dev server. Not running?"
thanks in advance :)

**Comment by jsjoeio** [comment]
I haven't used ssh-fs but if you're writing a React app and using the reverse proxy, you have to change a couple things. See [docs](https://github.com/cdr/code-server/blob/main/docs/FAQ.md#proxying-to-create-react-app). Let me know if that helps!

**Comment by wieluk** [comment]
Sorry, but to be honest I am not even sure what a React app is and I am not using a reverse proxy

**Reply by jsjoeio** [reply]
Interesting! Maybe a tool you're using is written in React?

What does your set up look like? Mind posting a video or a photos?
